### DISCUSSION TITLE
Welcome to Hello-World Discussions!

### DISCUSSION THREAD

**Discussion by Sid-WC121** [Sid-WC121/Hello-World#2] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by sukrith-sunil** [comment]
Assembly Language

**Reply by Sid-WC121** [reply]
nice

**Comment by sukrith-sunil** [comment]
Cobol

**Comment by msarkajkesavu** [comment]
Cobol language

**Comment by Sid-WC121** [comment]
### 
Some new languages

**Comment by nikz055** [comment, marked as answer]
Assembly language
